tsfile/src/test/java/org/apache/iotdb/tsfile/encoding/bitpacking/IntPackerTest.java

```diff
@@ -48,12 +44,12 @@ public void test() {
         bb[idx++] = tb[j];
       }
     }
-    int res[] = new int[count * 8];
-    packer.unpackAllValues(bb, 0, bb.length, res);
+    Integer res[] = new Integer[count * 8];
```

Comment:
why using Integer[] rather than int[]...

Reply:
These kinds of modifications will be revoked.